src/heap_utils.cc

```diff
@@ -0,0 +1,232 @@
+#include "node_internals.h"
```

Comment:
This does not seem to have to be in `src`? Can we create a directory in `test` that specifically build this as an addon so we don't need to build it into the binary and use `internalBinding` to load it?

Reply:
It does use `Environment::BuildEmbedderGraph` … would we want to expose that instead?